src/components/icons/Cross.tsx

```diff
@@ -0,0 +1,5 @@
+import {createSinglePathSVG} from './TEMPLATE'
+
+export const Cross = createSinglePathSVG({
```

Comment:
We have this icon already actually! Called `TimesLarge_Stroke2_Corner0_Rounded`. I'll link the icon file in Slack, has more info in there.